scripts/test/console_output_test.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
+
+import os
+import sys
+import pexpect
+
+def main():
+	if len(sys.argv) == 1:
+		print ("Usage: ./scripts/test/console_output_test.py <console_path>")
```

Comment:
"<console_path>" makes me think "cmake-build-debug/hyriseTest" instead of just "cmake-build-debug".

Call this the "<build_dir>" to make this more clear.